src/components/Header/Header.tsx

```diff
@@ -0,0 +1,140 @@
+import EES from "../../images/EES.svg";
+import MenuIcon from "../../images/EESMenuIcon.svg";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+interface Props {
+  screenWidth?: "mobile" | "desktop";
+}
+
+export function Header({ screenWidth }: Props) {
+  const [windowWidth, setWindowWidth] = useState<number>(window.innerWidth);
+  const [showMenuIcon, setShowMenuIcon] = useState<boolean>(false);
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+  const isMobile = screenWidth === "mobile";
+
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setWindowWidth(width); // Track window width
+      setShowMenuIcon(width < 600 ? true : false);
+    };
+    handleResize();
+    window.addEventListener("resize", handleResize);
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  return (
+    <>
+      <div
+        className={`flex justify-between items-center w-full h-[74px] px-[128px] lg:p-[64px] py-[80px] bg-wheat
+          border-x-4 border-transparent
+          ${
+            showMenu && windowWidth < 600
+              ? "border-x-lightGingerFlower border-x-4"
+              : ""
+          }
+          ${windowWidth < 600 ? "px-[8px] py-[8px]" : ""}
+          ${
+            isMobile
+              ? "px-[8px] py-[8px] border-x-lightGingerFlower border-x-4"
+              : "px-[128px] py-[80px]"
+          }
+          `}
+      >
+        <Link to="/">
+          {/* showMenu then add className with border-x-red */}
+          <img
```

Comment:
Ideally this would be actual text in an `h1` tag (better for SEO, screen-readers, can typically scale better) instead of an image.